src/pudl/output/ferc1.py

```diff
@@ -741,7 +744,7 @@ def add_mean_cap_additions(steam_df):
             .pipe(pudl.helpers.convert_cols_dtypes, "ferc1"),
             how="left",
             on=idx_steam_no_date,
-            validate="m:1",
+            validate="many_to_one",
```

Comment:
I standardized these merge validations throughout the file to use the readable but longer versions, and also made the left merges explicit.